subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3136,14 +3136,16 @@ static inline void event_len_prep(struct ll_conn *conn)
 		struct pdu_data_llctrl_length_req *lr;
 		struct pdu_data *pdu_ctrl_tx;
 		struct node_tx *tx;
+		/*
+		 * initialize to 0 to eliminate compiler warnings
+		 */
 		u16_t rx_time = 0;
 		u16_t tx_time = 0;
 		/*
-		 * Using bool instead of u8_t increases code size
-		 * in this case.
-		 */
-		u8_t feature_coded_phy;
-		u8_t feature_phy_2m;
+		** Using u32_t since we do a bitwise-and to check for features
```

Comment:
I dont see a bitwise-and?

Reply:
lines 3155 and 3162

Reply:
use !! to convert to boolean and assign to u8_t

feature_coded_phy = !!(conn->llcp_feature.features & BIT(BT_LE_FEAT_BIT_PHY_CODED));

Reply:
That does not give a reduction in code-size, and IMO does not increase readability. 
You still want me to make the change, given that code-size is unaltered?